src/WooCommerceServiceProvider.php

```diff
@@ -43,10 +43,11 @@ public function bindFilters()
         $woocommerce = $this->app['woocommerce'];
 
         add_filter('template_include', [$woocommerce, 'templateInclude'], 11);
-        add_filter('woocommerce_locate_template', [$woocommerce, 'template']);
-        add_filter('wc_get_template_part', [$woocommerce, 'template']);
+        add_filter('woocommerce_locate_template', [$woocommerce, 'template'], 10, 2);
+        add_filter('woocommerce_locate_core_template', [$woocommerce, 'template'], 10, 2);
+        add_filter('wc_get_template_part', [$woocommerce, 'template'], 10);
         add_filter('comments_template', [$woocommerce, 'reviewsTemplate'], 11);
-        add_filter('wc_get_template', [$woocommerce, 'template'], 1000);
+        add_filter('wc_get_template', [$woocommerce, 'template'], 1000, 2);
```

Comment:
```suggestion
        add_filter('wc_get_template', [$woocommerce, 'template'], 1000);
```

Since this is not being used.

Reply:
That's funny. In my testings, a plugin does not find templates within theme when I remove the seconds argument. Trying to figure out what's wrong, because the plugin hasn't changed (it's my own plugin in WP repo) and there were not compatibility issues pre-Sage 10.